dev-python/python-msrplib/Manifest

```diff
@@ -1 +1 @@
-DIST python-msrplib-0.18.0.tar.gz 30234 BLAKE2B da90d00291ab68ae5e4e89f275367741d8dac21e2cb35b6b9722430147b5ada1802c943b5e224f36d565faab216c649cc6e155fc1aaf23ab2f63d3c6e39bf2a7 SHA512 ed260b0183aff543e115951f464d0e3934862439a7c049c3d19304c543805ee364129d2a99c0d132561c8ae0af9fc19cde460723f2754e973bf68a602a18db99
+DIST release-0.19.1.tar.gz 30917 BLAKE2B 8b44b5c3e0383854a0e616cc3bf6fca937fd65ca1f44a7eee8b5647db5932acffdbbcf8c1433736d932ca50e2674203b0a2ad5f2c9858b83d65818357f7015b6 SHA512 afa05606354eb82cb7d1f5c95ac1a146c25c7d22b74a44b83625351b470646e91242468f3ee0a1ccb2b3193d3a222246a92a8133614ded4747493a1f6e2b193e
```

Comment:
Please split the removal and addition into separate commits, or preferably keep the old version for some time still.

Reply:
I recovered the former version on the new update